app/controllers/questions_controller.rb

```diff
@@ -128,6 +128,17 @@ def build_csv
     end
   end
 
+  def import
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def import_csv
+    Question.import(params[:file], current_user)
+    redirect_to questions_path, notice: "Upload Successful"
```

Comment:
Right notice doesn't show up on the question list page—you have to go to a page that is showing the notices. I think maybe I should add displaying of notices to the main layout or something. We can deal with that eventually.